### DISCUSSION TITLE
Unable to configure build on Windows

### DISCUSSION THREAD

**Discussion by centaurisean** [ngageoint/six-library#640] [Q&A]
I'm failing to build the *six-library* on Windows. Steps I've done so far:

1. Cloned repo. On **master** branch locally
2. Followed CMake instructions at https://github.com/ngageoint/six-library/blob/master/externals/coda-oss/cmake/README.md:
```
   # starting from base directory of repo, make a build directory and cd into it
   mkdir target
   cd target

   # configure
   cmake .. -DCMAKE_INSTALL_PREFIX=../install
```  

This gets a long way through the configure process, but ultimately fails with the following:
```
-- Configuring done
CMake Error at externals/coda-oss/modules/drivers/xml/xerces/CMakeLists.txt:281 (add_library):
  No SOURCES given to target: xerces-c
```

Other information:  

- Windows 10
- Visual Studio 2019
- CMake version 3.20.0-rc1
- Python version 3.11.0
  
Full build log here [cmake.log](https://github.com/ngageoint/six-library/files/11214181/cmake.log)

**Comment by JDanielSmith** [comment]
Try using the [main](https://github.com/ngageoint/six-library/tree/main) branch.

**Reply by centaurisean** [reply]
No change. I switched to the main branch, cleared out my *target* directory, and re-ran the ```cmake .. -DCMAKE_INSTALL_PREFIX=../install``` command. It still failed with:
```
-- Configuring done
CMake Error at externals/coda-oss/modules/drivers/xml/xerces/CMakeLists.txt:284 (add_library):
  No SOURCES given to target: xerces-c
```

I've attached the log again. Note at the top of the log I did a *git status* first so that you can verify I'm correctly on *main*.
[cmake_main.log](https://github.com/ngageoint/six-library/files/11214608/cmake_main.log)
